packages/ssr/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# ssr
```

Comment:
@silentworks not sure if this file is created if it does not exist, when we do the changeset stuff

Reply:
Yeah this gets created automatically by the changeset stuff.